### ISSUE TITLE
Missing alt text on some images

### ISSUE THREAD

**Issue by cupoftwee** [cupoftwee/sarahrobin#65]
Sometimes one needs to update a resume ASAP and forgets to circle back for a year or two. Time to fix it!
